src/components/checkbox/index.tsx

```diff
@@ -1,6 +1,8 @@
 import { h, Component } from 'preact';
+
+import { UncheckedIcon, CheckedIcon } from 'lib/icons';
+
```

Comment:
What's the logic behind the reordering and spacing in the changes?

Reply:
Basically, vendors go first, then anything from `src/lib` and `src/codecs`, then everything else, styles are always at the end, the groups are separated by an empty line. I think it makes code slightly easier to read when you don't have to scan every import declaration to find a certain import 🙂